Matchismo/Matchismo/Controller/AbstractCardGameViewController.h

```diff
@@ -9,20 +9,32 @@
 
 NS_ASSUME_NONNULL_BEGIN
 
-@class CardMatchingGame, Deck;
+@class Card, CardMatchingGame, CardView, Deck;
 
 @interface AbstractCardGameViewController : UIViewController
 
-@property (readonly, nonatomic) Deck *deck;
-@property (readonly, nonatomic) CardMatchingGame *game;
+@property (nonatomic) Deck *deck;
+@property (nonatomic) CardMatchingGame *game;
+@property (nonatomic) NSMutableArray<CardView *>* cardViews;
```

Comment:
Think I've said this before but it's usually a bad idea to expose mutable types as public properties. Much better to make the mutable array private and just have accessor methods for adding/removing views or whatever. 
Or the other option is to make it a settable `NSArray`.

Also, incidentally, having a mutable array this isn't `readonly` doesn't quite make sense - why would you ever need to set it if you can just call `removeAllObjects`?